### ISSUE TITLE
React Native doesn't use index.ios.js as the entry point anymore

### ISSUE THREAD

**Issue by fangpenlin** [Thinkmill/react-native-schemes-manager#25]
### Steps to reproduce the behavior

For iOS React Native app, change the entry point file from `index.ios.js` to `index.js`.

### Expected behavior

The build script should be able to find `index.js` when `index.ios.js` doesn't exist.

### Actual behavior

When the scrip is ran, no entry file is found

```
...
Cannot find entry file index.ios.js in any of the roots: ["/path/to/my/app"]
...
```

Since React Native 0.49

https://github.com/facebook/react-native/releases/tag/v0.49.0

They changed the example project to use `index.js` instead of two index files for each platform. It seems this is not really something required for newer RN version, it's just the default template changed. However, I think using `index.js` should also be fine, we may need to look into `index.ios.js` first, fallback to `index.js` if it's not found.

**Comment by blargity** [comment]
I suspect this will get resolved when #24 is merged.

**Comment by blargity** [comment]
@fangpenlin this should be resolved in 1.0.1. If that's not the case please open a new issue, but I'm pretty sure it is. Thanks for letting us know!
